scripts/img

```diff
@@ -23,4 +23,4 @@ ueberzugpp layer --no-stdin --silent --use-escape-codes --pid-file "$UB_PID_FILE
 UB_PID="$(cat "$UB_PID_FILE")"
 export UB_SOCKET=/tmp/ueberzugpp-"$UB_PID".socket
 ueberzugpp cmd -s "$UB_SOCKET" -a add -i PREVIEW -x "$1" -y "$2" --max-width "$3" --max-height "$4" -f "$5"
-read -r
+sleep 2
```

Comment:
Why this change?

Reply:
idk how this happened, but before i pushed my code the first time, during testing, `read -r` worked fine, but now it doesn't work anymore, ueberzugpp just exits immediately. idk what caused this, sorry

Reply:
Did that happen when you first uploaded the script?